frontend/vitest.setup.ts

```diff
@@ -17,13 +17,14 @@
 import * as polyfill from "polyfill-pseudoclass-has"
 import "vitest-canvas-mock"
 import { vi } from "vitest"
+import "@testing-library/jest-dom/vitest"
 
-// TODO(vite-update): This is a placeholder to avoid massive set of
-// changes since vi is mostly a drop-in replacement of jest
+// In the event a sub-library uses the jest global, we need to make sure it's
+// aliased to the vi global. An example is timers using react testing library
```

Comment:
Are there alternative libraries for that that use vitest instead of jest?

Reply:
As of now no, the associated issue is https://github.com/testing-library/dom-testing-library/issues/987 which is pulled in from the react-testing-library. Thankfully, vitest is mostly a drop-in replacement, so this makes things easier. I'll add this issue so we can follow it. I found this to be the easiest workaround. It looks like it's only used in the `waitFor` command. https://github.com/testing-library/dom-testing-library/blob/a86c54ccda5242ad8dfc1c70d31980bdbf96af7f/src/wait-for.js#L81